src/CribblyBackend.DataAccess/Services/TeamComparer.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections.Generic;
+using CribblyBackend.DataAccess.Models;
+
+namespace CribblyBackend.DataAccess.Services
+{
+    public class TeamComparer : IEqualityComparer<Team>
+    {
+        public bool Equals(Team team1, Team team2)
+        {
+            return team1.Id == team2.Id;
+        }
+
+        public int GetHashCode(Team team)
+        {
+            if(team == null) return 0;
+            unchecked
+            {
+                int hash = 69;
+                hash = (hash * 420) + team.Id;
+                return hash;
```

Comment:
wut?

Reply:
We need to make a hash value when telling the code how to compare teams so that ```.Distinct()``` works